WebService/v1/Models/SimulationListApiModel.cs

```diff
@@ -21,12 +21,5 @@ public SimulationListApiModel()
         {
             this.Items = new List<SimulationApiModel.SimulationApiModel>();
         }
-
-        /// <summary>Map a service model to the corresponding API model</summary>
-        public SimulationListApiModel(IEnumerable<Services.Models.Simulation> simulations)
```

Comment:
I'd prefer this approach to having this logic in the controller, it's part of the mapping logic

Reply:
:white_check_mark: